### ISSUE TITLE
Duplicates

### ISSUE THREAD

**Issue by aryeh-looker** [pirate/sites-using-cloudflare#8]
Lots of duplicates!

```
$ cat sorted_unique_cf.txt | wc -l
 7385121

$ cat sorted_unique_cf.txt | uniq |wc -l
 4287625
```

**Comment by pirate** [comment]
... odd, I ran uniq on it before compressing.  I guess I should've double checked!

**Comment by pirate** [comment]
Fixed in 1d388c3, uploading now.

**Comment by pirate** [comment]
Done: https://github.com/pirate/sites-using-cloudflare/raw/master/sorted_unique.zip
